spec/rails_helper.rb

```diff
@@ -255,7 +252,7 @@ def fab!(name, &blk)
 
     SiteSetting.provider = TestLocalProcessProvider.new
 
-    WebMock.disable_net_connect!(allow_localhost: true, allow: [Webdrivers::Chromedriver.base_url])
+    WebMock.disable_net_connect!(allow_localhost: true)
```

Comment:
I wonder whether this is part of the issue with the failing specs? Maybe we need to allow some kind of extra base_url?

Reply:
Hmm, no, looks like that's only used to update Webdrivers, `Webdrivers::Chromedriver.base_url` points to `https://chromedriver.storage.googleapis.com`.

Reply:
Yeah not sure about that yet, but mobile specs aren't getting the correct user_agent so trying to specify that first. Not seeing anything online regarding emulation for chromedriver changing however.